src/frontend/apps/standalone_site/src/styles/theme.extend.ts

```diff
@@ -37,4 +41,66 @@ export const themeExtend: ThemeType = deepMerge(theme, {
       },
     },
   },
-});
+  select: {
+    options: {
+      container: {
+        pad: 'small',
+      },
+      text: {
+        size: 'small',
+      },
+    },
+  },
+  formField: {
+    label: {
+      size: '0.8rem',
+      margin: '0.5rem 1rem 0',
+      color: 'bg-grey',
+    },
+    border: {
+      position: 'outer',
+      side: 'all',
+      color: 'blue-active',
+      style: 'solid',
+    },
+    round: {
+      size: 'xsmall',
+    },
+    margin: { bottom: 'large' },
+    extend: css`
+      & {
+        border: 1px solid;
+        border-bottom: 2px solid;
+        border-right: 2px solid;
+      }
+      & input::placeholder {
+        font-size: 0.8rem;
+        padding-left: 0.2rem;
+      }
+      & label {
+        margin-top: -9px;
+        background: white;
+        width: fit-content;
+        padding: 0 9px 0 5px;
+      }
+      &.mandatory label {
+        margin-top: -13px;
+      }
+    `,
+    textInput: {
+      extend: 'padding: 0 1rem 0.8rem',
+    },
+    maskedInput: {
+      extend: 'padding: 0 1rem 0.8rem',
+    },
+    dateInput: {
+      icon: {
+        size: '18px',
+      },
+    },
+  },
+};
+
+export const themeBase = theme;
+
+export const getFullThemeExtend = () => deepMerge(themeBase, themeExtend);
```

Comment:
I don't understand why you export the theme that is already exported in lib-common.
And I don't understant why you use a lambda to create the merged theme that should be called only once.

Reply:
`themeBase` is mainly to not have to search the theme in different package or module, if I want something about theme it is all here like that.